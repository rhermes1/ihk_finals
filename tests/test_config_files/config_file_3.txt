; *****************************************
; Viele Stroemungsgebiete und grosse Route
; *****************************************

; Groesse des Gebietes
70 86 

; zu fahrende Route

5 30 25 5 

; Stroemungen
0 0 35 30 1.0 -1.0
10 0 24 85 1.0 -1.0
40 40 65 43 3.0 -1.1
15 0 35 10 2.121 -2.121
0 0 65 15 1.79 0.894
15 10 25 30 0.0 0.0
15 0 35 10 2.121 -2.121
0 0 15 25 1.79 0.894
15 10 25 30 1.0 0.0
37 0 45 80 2.121 -2.121
65 0 70 86 8 -4 
15 10 25 30 1.0 0.0
15 0 35 10 2.121 -2.121
