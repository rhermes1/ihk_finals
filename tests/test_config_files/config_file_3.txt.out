****************************************
Viele Stroemungsgebiete und grosse Route
****************************************
Groesse des Gebietes
 70.0 86.0

Stroemungen
 (0.0;0.0) (70.0;86.0) SV = (0.0;0.0)

 (0.0;0.0) (35.0;30.0) SV = (1.0;-1.0)

 (10.0;0.0) (24.0;85.0) SV = (1.0;-1.0)

 (40.0;40.0) (65.0;43.0) SV = (3.0;-1.1)

 (15.0;0.0) (35.0;10.0) SV = (2.1;-2.1)

 (0.0;0.0) (65.0;15.0) SV = (1.8;0.9)

 (15.0;10.0) (25.0;30.0) SV = (1.0;0.0)

 (0.0;0.0) (15.0;25.0) SV = (1.8;0.9)

 (37.0;0.0) (45.0;80.0) SV = (2.1;-2.1)

 (65.0;0.0) (70.0;86.0) SV = (8.0;-4.0)

zu fahrende Route
 Punkt 0: (5.0;30.0)

 Punkt 1: (25.0;5.0)


Daten fuer die Planung der Route:
 Strecke von (5.0;30.0) nach (9.0;25.0)
  Entfernung: 6.4 km
  SKV = (2.1;-2.9)
  Sollgeschwindigkeit: 3.6 km/h
  Fahrzeit: 1.8 h

 Strecke von (9.0;25.0) nach (15.0;17.5)
  Entfernung: 9.6 km
  SKV = (1.3;-4.8)
  Sollgeschwindigkeit: 5.0 km/h
  Fahrzeit: 1.9 h

 Strecke von (15.0;17.5) nach (21.0;10.0)
  Entfernung: 9.6 km
  SKV = (2.1;-3.9)
  Sollgeschwindigkeit: 4.4 km/h
  Fahrzeit: 2.2 h

 Strecke von (21.0;10.0) nach (25.0;5.0)
  Entfernung: 6.4 km
  SKV = (1.3;-4.8)
  Sollgeschwindigkeit: 5.0 km/h
  Fahrzeit: 1.3 h
